pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -102,12 +130,26 @@ def load_supported_locales
 
 def language_to_locale(language)
```

Comment:
Can we have a similar comment here that this is for converting hourofcode.com's 2 letter language codes to pegasus/apps 4 letter locale codes?